### DISCUSSION TITLE
v5.66.0

### DISCUSSION THREAD

**Discussion by sokra** [webpack/webpack#15151] [General]
# Features

* add `output.library.type: "commonjs-static"` to emit a statically analyse-able commonjs module (for node.js esm interop support)
* add `experiments.css` (very experimental)
  * see https://github.com/webpack/webpack/issues/14893

# Bugfixes

* fix CORS headers for `experiments.lazyCompilation`
* fix `[absolute-resource-path]` for SourceMap module naming
* avoid stack overflow when accessing many memory cached cache values in series

# Performance

* reduce default `watchOptions.aggregateTimeout` to 20ms

<hr /><em>This discussion was created from the release <a href='https://github.com/webpack/webpack/releases/tag/v5.66.0'>v5.66.0</a>.</em>

**Comment by barak007** [comment]
Where can we talk about the new built-in css support? 
The issue is blocked for comments.

**Reply by alexander-akait** [reply]
It is still experiential and can be unstable, you can use discussions to report bugs, we will unlock it when it was more mature

**Reply by bilygc** [reply]
i cant build any new project since this version cause says: TypeError: MiniCssExtractPlugin is not a constructor
